server.js

```diff
@@ -4,13 +4,18 @@ const session = require("express-session")
 const bodyParser = require("body-parser")
 require('dotenv').config()
 const cookieParser = require("cookie-parser")
-const bcrypt = require('bcrypt');
+const ObjectId = require('mongoose').Types.ObjectId;
 const { Schema,
     userSchema,
     User,
     connect,
     sessionSchema,
-    Session } = require("./users.js");
+    Session,
+    Post } = require("./database.js");
```

Comment:
I know we said it's lower priority, but it's getting really hard to follow what imports are used and what aren't. Do try to clean up unused imports before PRing